packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -119,23 +119,23 @@ export function FeedbackButton(inProps: FeedbackButtonProps) {
 
     const tooltip = (
         <Tooltip
-            title={displayState === "error" ? tooltipErrorMessage : tooltipSuccessMessage}
+            title={displayState === "error" ? tooltipErrorMessage : displayState === "success" ? tooltipSuccessMessage : ""}
             open={displayState === "error" || displayState === "success"}
             placement={endIcon && !startIcon ? "top-end" : "top-start"}
             variant={resolveTooltipForDisplayState(displayState)}
             {...slotProps?.tooltip}
         >
-            <span>{startIcon ? startIcon : endIcon}</span>
+            <span>{startIcon || endIcon}</span>
         </Tooltip>
     );
 
     return (
         <Root
             ownerState={ownerState}
-            loading={loading}
+            loading={loading !== undefined ? loading : displayState === "loading"}
```

Comment:
This shows why having state duplication isn't a good idea 😁 Ideally, the FeedbackButton wouldn't have any internal states

Reply:
Jup ^^ Not my idea though. I just made it work properly with how the component was designed initially

Reply:
@thomasdax98 we could think about changing this API in v8